lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -361,7 +361,6 @@ def define
       outcome :outcome_opposite_sex_marriage_in_south_korea
       outcome :outcome_opposite_sex_marriage_in_slovakia
       outcome :outcome_opposite_sex_marriage_in_thailand
-      outcome :outcome_opposite_sex_marriage_in_turkey
```

Comment:
It seems that If this is removed in the following commit, the tests would not be green.